package.json

```
@@ -289,10 +289,10 @@
     "typescript": "4.5.5"
   },
   "dependencies": {
+    "@7sean68/jacoco-parse": "^2.2.0",
```

Probably just ping this to `2.2.0` instead.
```suggestion
    "@7sean68/jacoco-parse": "2.2.0",
```